hack/extract-kubeconfig.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+namespace="${1:-}"
+test="${2:-}"
+if [[ -z "${namespace}" || -z "${test}" ]]; then
+	echo "USAGE: $0 <namespace> <test>"
+	exit 1
+fi
+
+echo "Scanning ${namespace} for the \$KUBECONFIG for the $test test..."
+raw="$( oc get secrets --namespace "${namespace}" -o jsonpath="{.items[?(@.metadata.name==\"${test}\")].data.kubeconfig}" )"
+if [[ -n "${kubeconfig}" ]]; then
+	output="$( mktemp /tmp/kubeconfig.XXXXX )"
+	echo -n "${kubeconfig}" | base64 --decode > "${output}"
+	echo "\$KUBECONFIG saved to ${output}"
```

Comment:
```sh
echo "export KUBECONFIG='${output}'"
```

so folks can `eval $(hack/extract-kubeconfig.sh)`?  You'd have to write the rest of the script's output to stderr instead (e.g. the earlier usage message), but it would make it more like `ssh-agent` and similar.

Reply:
My `$KUBECONFIG` is a special thing I'm pretty careful with, so I don't know that most people want that? Maybe that's just because I'm logged into N prod clusters usually